var/spack/repos/builtin/packages/dtc/package.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Dtc(MakefilePackage):
```

Comment:
(no changes required) FYI I'm fine with adding new packages to a PR if they are mentioned in some way by a dependency. In this case though this package appears to be totally separate: not mentioned by any of the other packages (and likewise this doesn't mention any of the other packages in this PR); normally I think such a package should be added in a separate PR.

Reply:
The commit message includes this line.
`- add dtc (dependency for riscv-isa-sim which may be added later)`

Since the intermediate package is missing there is no dependence in this PR.